src/containers/FileDetailContainer.js

```diff
@@ -0,0 +1,66 @@
+import React, {useState} from 'react';
+import {withRouter} from 'react-router-dom';
+import FileDetail from '../components/FileDetail/FileDetail';
+import {LoadingPlaceholder} from '../components/Loading';
+import {Query, Mutation} from 'react-apollo';
+import {UPDATE_FILE} from '../state/mutations';
+import {GET_FILE_BY_ID} from '../state/queries';
+import PropTypes from 'prop-types';
+
+const FileDetailContainer = ({kfId, history, match}) => {
+  const [fileNameInput, setFileName] = useState();
```

Comment:
Why do you only need to track the name state and not description and fileType here?